core/config.cpp

```diff
@@ -77,15 +78,27 @@ bool Config::init_with_args(int ac, char** av, const std::vector<std::string>& c
                                      "Worker information.\nFormat is '%worker_hostname:%thread_number'.\nUse "
                                      "colon ':' as separator.");
 
+    po::options_description optional_options("Optional options");
```

Comment:
you mean put it into work info part. And user specify the memory size and page size for each worker respectively?